pkg/launch/launchable.go

```diff
@@ -59,10 +59,13 @@ type LaunchableStanza struct {
 	// that should be implemented by the pod itself via bin/post-activate
 	NoHaltOnUpdate bool `yaml:"no_halt_on_update,omitempty"`
 
-	// Specifies which files or directories (relative to launchable root)
-	// should be launched under runit. Only launchables of type "hoist"
-	// make use of this field, and if empty, a default of ["bin/launch"]
-	// is used
+	// In the case of hoist launchables, specifies which files or
+	// directories (relative to launchable root) should be launched under
+	// runit. if empty, a default of ["bin/launch"] is used.
+	//
+	// In the case of docker containers, it specifies which commands
+	// should be started in the docker container. If none are specified, the
+	// docker image's ENTRYPOINT is used.
```

Comment:
Perhaps this is best flushed out in a design doc, but what do we do about pods following the Hoist convention of having multiple launch scripts? Listing entry points here is one option, but that requires synchronizing this list to what's actually deployed (as opposed to "run everything in this directory"). Another option (perhaps not mutually exclusive) might be to have a single Docker entry point that runs a supervisor within the container (e.g., runit).

Thoughts?

Reply:
I meant to roll back this comment change actually, because I didn't end up using this.

I wonder if we can just not support overriding the entry point at all and use whatever is in the image?